### DISCUSSION TITLE
Add the way to get achievement Starstruck for all repository owners

### DISCUSSION THREAD

**Discussion by Ar4ikov** [community/community#53636] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello! I've noticed that Starstruck achievement can be obtained only by original repo creator. But it seems like all repository owners can obtain it in my POV.

For the honest obtainability, I'm suggesting you these 3 simple rules: 
1. Only owners that had beed added to repo before 16/128/512/4096 stars at repo (for every step) can obtain Starstruck
2. Only owners with some activity (by time, by commits count, by something else count) can obtain Starstruck
3. _(not really nessesary)_ For organizations, all of the owners of organization can obtain Starstruck (if 2 rules above have been completed)

After these changes, at least some of the developers who really, really want to get this achievement will be satisfied 😄

**Comment by YasinDehfuli** [comment]
The starstruck badge just gave to repository owner, and other members can't get the Starstruck Achievement.

But You can get [Pair Extraordinaire](https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.pair-extraordinaire.md) Badge With your Teammates, and friends, Here is course to learn you how to get Pair Extraordinaire badge : 
Use It to get this badge with your friend :)

https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.pair-extraordinaire.md

**Reply by ARHAM5432** [reply]
my repository has 18 badges but i didn't get the starstruck badge

**Reply by versatility519** [reply]
OMG

**Reply by Istituto-freudinttheprodev** [reply]
@senorL
@versatility519
@ARHAM5432
star mine i stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Reply by durjoybarua5327** [reply]
https://github.com/durjoybarua5327?tab=repositories

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Reply by AncientYoshi** [reply]
https://github.com/AncientYoshi/Video-Enhancer-AI | can any one star me

**Comment by AGK-1** [comment]
Only owners that had beed added to repo before 16/128/512/4096 stars at repo (for every step) can obtain Starstruck

**Reply by YasinDehfuli** [reply]
> Only owners that had beed added to repo before 16/128/512/4096 stars at repo (for every step) can obtain Starstruck

Create a ticket to [GitHub support](https://support.github.com/request) ,they will fix your achievement if your repo got enough stars

ف

**Reply by Istituto-freudinttheprodev** [reply]
@AGK-1
@YasinDehfuli
@AyushKatre05
star mine i stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Reply by durjoybarua5327** [reply]
https://github.com/durjoybarua5327?tab=repositories

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by dr4g0n1x** [comment]
Let's exchange stars
https://github.com/dr4g0n1x/ds-self-host

**Reply by Istituto-freudinttheprodev** [reply]
@dr4g0n1x
star mine i stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Reply by khalifarsm** [reply]
https://github.com/khalifarsm/botox-android

**Reply by whatsupsumit** [reply]
https://github.com/whatsupsumit/The-Nexus
star my repo i will star yours....
😊😊

**Reply by Finnimonius** [reply]
https://github.com/Finnimonius/react-ts-ai-rpg

**Reply by durjoybarua5327** [reply]
https://github.com/durjoybarua5327?tab=repositories

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by khalifarsm** [comment]
https://github.com/khalifarsm/botox-android

**Comment by KrrishSR4** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> Hello! I've noticed that Starstruck achievement can be obtained only by original repo creator. But it seems like all repository owners can obtain it in my POV.
> 
> For the honest obtainability, I'm suggesting you these 3 simple rules:
> 
> 1. Only owners that had beed added to repo before 16/128/512/4096 stars at repo (for every step) can obtain Starstruck
> 2. Only owners with some activity (by time, by commits count, by something else count) can obtain Starstruck
> 3. _(not really nessesary)_ For organizations, all of the owners of organization can obtain Starstruck (if 2 rules above have been completed)
> 
> After these changes, at least some of the developers who really, really want to get this achievement will be satisfied 😄

Star this https://github.com/KrrishSR4/KrrishSR4

**Reply by KrrishSR4** [reply]
Done

you should also star my repo..

**Reply by KrrishSR4** [reply]
Thanks bro aapka profile dekha aap bhot tagde lg rhe ho.. 🔥

**Comment by YarinsHyper** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> Hello! I've noticed that Starstruck achievement can be obtained only by original repo creator. But it seems like all repository owners can obtain it in my POV.
> 
> For the honest obtainability, I'm suggesting you these 3 simple rules:
> 
> 1. Only owners that had beed added to repo before 16/128/512/4096 stars at repo (for every step) can obtain Starstruck
> 2. Only owners with some activity (by time, by commits count, by something else count) can obtain Starstruck
> 3. _(not really nessesary)_ For organizations, all of the owners of organization can obtain Starstruck (if 2 rules above have been completed)
> 
> After these changes, at least some of the developers who really, really want to get this achievement will be satisfied 😄

Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼
